wm-system/src/main/kotlin/com/ajaxproject/warehouse/service/ProductServiceImpl.kt

```diff
@@ -17,33 +17,33 @@ import reactor.core.publisher.Mono
 @Service
 @Validated
 class ProductServiceImpl(
-    val productRepository: ProductRepository
+    val productCacheableRepository: ProductCacheableRepository
```

Comment:
let's not create a separate interface. you can have 1 interface ProductRepository and two implementations

Reply:
done